tests/autologging/test_autologging_utils.py

```diff
@@ -466,7 +466,8 @@ def test_autologging_integration_forwards_positional_and_keyword_arguments_as_ex
     def autolog(foo=7, bar=10, disable=False, silent=False):
         return foo, bar, disable
 
-    assert autolog(1, bar=2, disable=True) == (1, 2, True)
+    assert autolog(1, bar=2, disable=True) is None
```

Comment:
I think we can remove this assert, as it isn't really testing the behavior being described in the test case.